### PR TITLE
fix 'Wrong behaviour for window.fetch request called without parameters' (close #939)

### LINKED ISSUE #939: Wrong behaviour for window.fetch request called without parameters
Found in http://bxt-to-wc-weu-tst.azurewebsites.net/ site.

Without proxing:
```js
var result = window.fetch();
result:
[[PromiseStatus]]:"rejected"
[[PromiseValue]]:TypeError: Failed to execute 'fetch' on 'Window': 1 argument required, but only 0 present. at eval (eval at evaluate (:117:21), <anonymous>:1:13) at http://localhost:3000/:16:5
```

With proxing:
```js
var result = window.fetch();
result:
[[PromiseStatus]]:"pending"
[[PromiseValue]]:undefined
```

### PR DESCRIPTION
**Pull request by miherlosev**
@churkin @LavrovArtem

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
rebase please

**Comment by miherlosev** [comment]
done

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR

File: src/client/sandbox/fetch.js [hunk 1]
```diff
@@ -42,7 +42,13 @@ export default class FetchSandbox extends SandboxBase {
         args[1] = FetchSandbox._processRequestInit(init);
     }
 
+    static _isValidRequestArgs (args) {
+        return typeof args[0] === 'string' || isFetchRequest(args[0]);
+    }
```
Comment (LavrovArtem) [inline src/client/sandbox/fetch.js]: Add new line after this function

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR

**Pull request merged**